bin/test_integration_rhel

```diff
@@ -1,23 +1,32 @@
 #!/bin/bash -ex
 
+CONTAINER_ID="$1"
```

Comment:
now that we have more than one input, can we change this to pure flags? 

something like this will do:
```
while true ; do
  case "$1" in
    --rhel-version=* ) RHEL_VERSION="${option#*=}" ; shift ;;
    --debug ) DEBUG="true" ; shift ;;
    --help ) print_help ; shift ;;
     * ) if [ -z "$1" ]; then break; else echo "$1 is not a valid option"; exit 1; fi;;
  esac
done
```

Note that i changed `CONTAINER_ID` to `RHEL_VERSION` to be more accurate (as i understand it) and that i added a `print_help` call that you may implement if you like. something like this will do (but change the data):
```
function print_help() {
  cat << EOF

Creates an Appliance release

Usage: start [options]

    -h, --help        Shows this help message.

    -r, --revert      Removes the current version tags. This allows the appliance
                      to be re-released.

    -f, --force       Release even if there are no changes. This is necessary
                      after a revert.

EOF
}
```

Reply:
regarding the `CONTAINER_ID` being `test-rhel7/8`, i think it is an implementation detail. i would get `RHEL_VERSION=7/8` as input and set `CONTAINER_ID="test-rhel${RHEL_VERSION}` once we get it from the user.

Reply:
Plus plus on this